lib/autoproj/ops/configuration.rb

```diff
@@ -157,10 +157,10 @@ def create_remote_set_user_dir(vcs)
                 symlink_dest
             end
 
-            def load_package_set(vcs, options, imported_from)
+            def load_package_set(vcs, options, imported_from, initrb: true)
```

Comment:
It seems that `load_package_set` is never used outside of `load_and_update_package_sets`. Now that you load init.rb within `update_package_sets`, the loading here should be removed.